### ISSUE TITLE
Bug report command

### ISSUE THREAD

**Issue by petrspelos** [discord-bot-tutorial/Community-Discord-BOT#97]
:beetle: A simple `bug` or `bug-report` command should be implemented that would point to:

https://github.com/discord-bot-tutorial/Community-Discord-BOT/issues/new/choose

:bulb: Alias could be `issue`, `request`, `feedback`, since there are options for those as well.

```
User: @Miunie bug // or bug-report, issue, request, feedback, ... perhaps more

Miunie: Thank you for your interest, how about you let us know by creating an Issue here:
https://github.com/discord-bot-tutorial/Community-Discord-BOT/issues/new/choose
```

***
:information_source: Interested in this issue? Why not give it a try and create a pull request? :blush: Anyone is welcomed to contribute! :tada:

**Comment by TheSecurity** [comment]
I will try to do it

**Comment by petrspelos** [comment]
Fixed with 28dbec42f176bad67fbd993cd052c0d7034639fe
